src/pivot-table/hooks/use-cell-height.ts

```diff
@@ -0,0 +1,35 @@
+import { useMemo } from "react";
+import { LayoutService, StyleService } from "../../types/types";
+import { DEFAULT_ROW_HEIGHT } from "../constants";
+
+interface UseCellHeightProps {
+  styleService: StyleService;
+  layoutService: LayoutService;
+}
+
+interface UseCellHeight {
+  (args: UseCellHeightProps): {
+    lineClamp: number;
+    headerCellHeight: number;
+    contentCellHeight: number;
+  };
+}
+
+const LINE_HEIGHT_COEFFICIENT = 4 / 3;
+const fontSizeToRowHeight = (fontSize: string, lineClamp: number) =>
```

Comment:
I think you need to include cell padding here. I did some testing and with characters like "g", it can get clipped at the bottom.